modules/private/hook.go

```diff
@@ -94,6 +95,10 @@ type HookProcReceiveRefResult struct {
 	IsForcePush  bool
 	IsNotMatched bool
 	Err          string
+	CreatePR     bool
+	URL          string
+	Message      bool
```

Comment:
Strange variable names. Above: `IsXxx bool`, but here: `CreatePR bool`, `Message bool` ?

I think they should be: `IsCreatePR` and `ShouldShowMessage`

Reply:
2f0f7ce